common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -248,7 +248,8 @@ def get_html(self):
             'submitted': self.submitted,
             'instructions_default': WordCloudFields.instructions.default,
             'instructions': self.instructions,
-            'display_name': self.display_name
+            'display_name': self.display_name,
+            'display_name_default': WordCloudFields.display_name.default
```

Comment:
Sorry to mention this now, but wouldn't it be easier to not pass the defaults to the HTML, and instead take into account the default values when display_name and instructions are passed?

````
'instructions': if self.instructions is not WordCloudFields.instructions.default self.instructions else None
````

Then the HTML check is simplified to whether the element is None.